content/en/open-source.md

```diff
@@ -45,6 +45,32 @@ Here is an overview of important repositories that we have created—directly or
     <a class="button button-primary read-more-button" href="https://github.com/datenanfragen/data">View repository&nbsp;<span class="icon icon-arrow-right"></span></a>
 </article>
 
+<article class="list-article icon-list-article">
+    <div class="col25 article-featured-image"><a href="https://github.com/datenanfragen/media"><img class="image" src="/card-icons/view.svg" alt="media"></a></div>
+    <div class="padded col75">
+        <a href="https://github.com/datenanfragen/media"><h1><code>media</code></h1></a>
+        <span class="license">MIT License</span>
+        <p class="description">
+            Our website used for hosting videos, media.datenanfragen.de. This repository includes the source code and scripts used for the site.
+        </p>
+    </div>
+    <div class="clearfix"></div>
+    <a class="button button-primary read-more-button" href="https://github.com/datenanfragen/media">View repository&nbsp;<span class="icon icon-arrow-right"></span></a>
+</article>
+
+<article class="list-article icon-list-article">
+    <div class="col25 article-featured-image"><a href="https://github.com/datenanfragen/letter-generator"><img class="image" src="/card-icons/letter.svg" alt="letter-generator"></a></div>
+    <div class="padded col75">
+        <a href="https://github.com/datenanfragen/letter-generator"><h1><code>letter-generator</code></h1></a>
+        <span class="license">MIT License</span>
+        <p class="description">
+            A tool that allows you to generate letters (plain text or PDF formats) from pre-defined templates. This repository includes these templates and the scripts needed to render letters from them.
```

Comment:
I think "templates" could easily be confused with the sample letters in `data` (I admit that this is our fault—we named them both the same :D). Maybe "layout templates" is clearer?